src/components/image/content/Image.tsx

```diff
@@ -1,20 +1,18 @@
 import * as React from 'react';
-import { StyleImage } from '../styles/image.styles';
+import { ImageContainer, StyledImage } from '../styles/image.styles';
 import { ImageProps } from './image_types';
 
 const Image: React.FC<ImageProps> = ({ src, ...props }: ImageProps) => {
   return (
-    <>
-      <StyleImage src={src} {...props} />
-    </>
+    <ImageContainer shape={props.shape} width={props.width}>
+      <StyledImage src={src} {...props} />
+    </ImageContainer>
   );
 };
+
 Image.defaultProps = {
-  src: `https://picsum.photos/100/200`,
```

Comment:
plz add default pic 

https://us-wbe-img.gr-cdn.com/user/8a9c5bd8-0d48-4f74-839c-04b16e7e404a/610efd88-ae0d-48b7-96fb-8bc45cc8ad61.png

if error hapened on loading image or src is null

Reply:
bayad sharte khassi bezaram ya default bezaram faghat okeye ??